qcfractal/storage_sockets/sql_models.py

```diff
@@ -367,6 +384,19 @@ class ResultORM(BaseResultORM):
     # schema_name = Column(String)  # default="qc_ret_data_output"??
     # schema_version = Column(Integer)
 
+    __table_args__ = (
+        # TODO: optimize indexes
+        # A multicolumn GIN index can be used with query conditions that involve any subset of
+        # the index's columns. Unlike B-tree or GiST, index search effectiveness is the same
+        # regardless of which index column(s) the query conditions use.
+        # Index('ix_result_combined', "program", "driver", "method", "basis",
+        #       "keywords", postgresql_using='gin'),  # gin index
+        # Index('ix_results_molecule', 'molecule'),  # b-tree index
+
+        UniqueConstraint("program", "driver", "method", "basis",
```

Comment:
We find that we often have lookups that look ups where the full 6 indices are not provided. Does a single multi-index provide good lookup speeds for just `program=rdkit`?

Reply:
The B-tree index will still be used but not efficiently if the key is on the far right. Therefore,  I marked this as todo, so that we order the 6 keys where the ones on the left are the most used query keys.